lib/plug/conn/utils.ex

```diff
@@ -293,4 +299,16 @@ defmodule Plug.Conn.Utils do
 
   defp downcase_char(char) when char in @upper, do: char + 32
   defp downcase_char(char), do: char
+
+  defp split_unquoted(bin, s, groups \\ [<<>>], quoted? \\ false)
+  defp split_unquoted(<<>>, _s, groups, _quoted?),
+    do: groups
+  defp split_unquoted(<<?", t :: binary>>, s, [g | groups], quoted?),
+    do: split_unquoted(t, s, [<<g :: binary,?">> | groups], !quoted?)
+  defp split_unquoted(<<h, t :: binary>>, s, groups, false) when <<h>> == s,
+    do: split_unquoted(t, s, [<<>> | groups], false)
+  defp split_unquoted(<<h, t :: binary>>, s, [g | groups], true) when <<h>> == s,
+    do: split_unquoted(t, s, [<<g :: binary, ?;>> | groups], true)
```

Comment:
Well I guess you don't need this function head, since it does exactly the same as the last function head.